src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/split_fc.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/cpu_opset/common/op/fully_connected.hpp"
+#include "split_fc.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include <transformations/utils/utils.hpp>
+#include "openvino/opsets/opset12.hpp"
+
+#include "itt.hpp"
+
+/*
+    -----       -----------       -----------
+    | A |   x   |    B    |   =   |    C    |
+    -----       -----------       -----------
+                    |   |   |
+                    V   V   V
+    -----       -----------       -----------
+    | A |   x   | B0 | B1 |   =   | C0 | C1 |
+    -----       -----------       -----------
+*/
+ov::intel_cpu::SplitFC::SplitFC() {
```

Comment:
@v-Golubev could you please take a look on the transformation? The goal is to have generic pass that splits FC (even with dynamic shapes) by output channels on several parallel branches